README.md

```diff
@@ -54,32 +60,73 @@ $ git clone https://github.com/IBM/casis-satellite-tracker
 
 ### 2. Create Watson services with IBM Cloud
 
-Create the following services:
+Create the [*Watson Assistant*](https://console.ng.bluemix.net/catalog/services/conversation) service by providing a name of your choice and clicking `Create`.
+
+Once created, you'll see the credentials for *username* and *password* that you should copy down to be used later. (Click `Show` to expose them).
 
-* [**Watson Assistant**](https://console.ng.bluemix.net/catalog/services/conversation)
+![](doc/source/images/WatsonAssistantCredentials.png)
 
-### 4. Import the Conversation workspace
+### 3. Import the Watson Assistant workspace
 
-Launch the **Watson Conversation** tool. Use the **import** icon button on the right
+Once you have created your instance of Watson Assistant, click `Launch Tool`, then click the `Workspaces` tab. Import the workspace by clicking the upload icon:
 
-Find the local version of [`data/workspaces/satelliteChat.json`](data/workspaces/satelliteChat.json) and select
-**Import**. Find the **Workspace ID** by clicking on the context menu of the new
-workspace and select **View details**. Save this ID for later.
+![](doc/source/images/UploadWorkspaceJson.png)
 
-*Optionally*, to view the conversation dialog select the workspace and choose the
-**Dialog** tab, here's a snippet of the dialog:
+Click `Choose a file` and navigate to [`data/workspaces/SatTrackWorkspace.json`](data/workspaces/SatTrackWorkspace.json) in this repo. Click `Import`.
 
-![](doc/source/images/dialog.PNG)
+Get the Workspace ID by clicking the 3 vertical dots on the `Workspaces` tab. Save this for later.
 
-### 5. Configure credentials
+<p align="center">
+  <img width="200" height="300" src="doc/source/images/GetAssistantDetails.png">
+</p>
+
+### 4. Get the Watson Assistant credentials
 
 The credentials for IBM Cloud Watson Assistant service can be found
-by selecting the ``Service Credentials`` option for each service.
+by selecting the ``Service Credentials`` option for the service. You
+saved these in [step #2](#2-create-watson-services-with-ibm-cloud).
+
+The `WORKSPACE_ID` for the Watson Assistant workspace was saved in
+[step #3](#3-import-the-watson-assistant-workspace).
+
+### 5. Create a Node-RED Workspace
+
+From the the [IBM Cloud Catalog](https://console.bluemix.net/catalog/) navigate to `Platform` -> `Boilerplates` and choose [Node-RED Starter](https://console.bluemix.net/catalog/starters/node-red-starter). Choose a name and click `Create`.
+
+Once the App has deployed, click on `Visit App URL`
+
+![](doc/source/images/NodeRedVisitAppURL.png)
+
+Follow the instructions to `Secure your Node-RED editor` and `Browse available IBM Cloud nodes`. Click `Finish` and then click `Go to your Node-RED flow editor`.
+
+### 6 Install additional nodes
+
+You will need to install the following additional nodes:
+
+```
+[node-red-contrib-credentials](https://flows.nodered.org/node/node-red-contrib-credentials)
```

Comment:
i think these are supposed to be hyperlinks? remove the backticks and make it a bullet list. It would be nice to give the reader some guidance here, but i've never done this before so maybe it's just that easy to follow.

Reply:
Yes, I'll add some detailed instructions in another patch.